src/main/java/ch/njol/skript/conditions/CondIsSaddled.java

```diff
@@ -0,0 +1,42 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.entity.AbstractHorse;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Steerable;
+import org.bukkit.inventory.ItemStack;
+
+public class CondIsSaddled extends PropertyCondition<LivingEntity> {
+
+	static {
+		register(CondIsSaddled.class, "[exact:(exactly|properly)] saddled", "livingentities");
+	}
```

Comment:
I'm a little confused on what this condition is actually doing especially when using exactly or properly. I see that we're checking for a saddle but it seems weird to be doing this

Reply:
The `exact|properly` bit is just for seeing if the horse's saddle is actually a saddle. Horses can have any item set as their saddle (albeit not via Skript or vanilla gameplay), but no matter the item in the saddle slot, they will be rideable.

Reply:
Yeah, the only way to set the saddle of a horse to anything but a saddle is by calling `AbstractHorseInventory#setSaddle(ItemStack)`. So yeah, niche for sure, I just thought it'd be worth handling just in case. Happy to remove it if the consensus is that it's essentially useless.